media-gfx/hydrus/hydrus-441.ebuild

```
@@ -0,0 +1,154 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..8} )
+PYTHON_REQ_USE="sqlite"
+
+inherit python-single-r1 desktop
+
+DESCRIPTION="A booru-like media organizer for the desktop"
+HOMEPAGE="http://hydrusnetwork.github.io/hydrus/ https://github.com/hydrusnetwork/hydrus"
+SRC_URI="https://github.com/hydrusnetwork/${PN}/archive/v${PV}.tar.gz -> ${PF}.tar.gz"
```

```suggestion
SRC_URI="https://github.com/hydrusnetwork/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

It won't change on revisions as we derive the URL from ${PV} anyway.